lib/algolia.ex

```diff
@@ -558,7 +558,7 @@ defmodule Algolia do
   end
 
   @doc """
-  Convinient version of wait_task/4, accepts a response to be waited on
+  Convenient version of wait_task/4, accepts a response to be waited on
```

Comment:
This is definitely an improvement but since we intend to contribute this upstream we should be even stricter about extraneous changes